Cargo.toml

```diff
@@ -107,6 +107,7 @@ prost = "0.12"
 raft-engine = { git = "https://github.com/tikv/raft-engine.git", rev = "22dfb426cd994602b57725ef080287d3e53db479" }
 rand = "0.8"
 regex = "1.8"
+regex-automata = { version = "0.1", features = ["transducer"] }
```

Comment:
Why do we use such an old version instead of the latest 0.4.x version?
https://crates.io/crates/regex-automata/0.4.3

Reply:
We need FST Transducer, which is not provided in the latest version. Here is some discussion: https://github.com/rust-lang/regex/issues/1087